core/spec/lib/spree/core/importer/order_spec.rb

```diff
@@ -403,86 +403,96 @@ module Core
         }.to raise_error /XXX/
       end
 
       it 'build a source payment using years and month' do
-        params = { :payments_attributes => [{
-                                              amount: '4.99',
-                                              payment_method: payment_method.name,
-                                              status: 'completed',
-                                              source: {
-                                                name: 'Fox',
-                                                last_digits: "7424",
-                                                cc_type: "visa",
-                                                year: '2022',
-                                                month: "5"
-                                              }
-                                            }]}
+        params = { payments_attributes: [{
+                                           amount: '4.99',
+                                           payment_method: payment_method.name,
+                                           status: 'completed',
+                                           source: {
+                                             name: 'Fox',
+                                             last_digits: "7424",
+                                             cc_type: "visa",
+                                             year: '2022',
+                                             month: "5"
+                                           }
+                                         }]}
 
         order = Importer::Order.import(user, params)
         expect(order.payments.first.source.last_digits).to eq '7424'
       end
 
       it 'handles source building exceptions when do not have years and month' do
-        params = { :payments_attributes => [{
-                                              amount: '4.99',
-                                              payment_method: payment_method.name,
-                                              status: 'completed',
-                                              source: {
-                                                name: 'Fox',
-                                                last_digits: "7424",
-                                                cc_type: "visa"
-                                              }
-                                            }]}
+        params = { payments_attributes: [{
+                                           amount: '4.99',
+                                           payment_method: payment_method.name,
+                                           status: 'completed',
+                                           source: {
+                                             name: 'Fox',
+                                             last_digits: "7424",
+                                             cc_type: "visa"
+                                           }
+                                         }]}
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.<br>Space inside } missing.